examples/multicsc/plot_sample_evoked_response.py

```diff
@@ -40,7 +40,7 @@
 ###############################################################################
 # Next, we define the parameters for multivariate CSC
 
-from alphacsc import GreedyCDL
+from alphacsc import GreedyCDL # noqa
```

Comment:
Why do you need to add `#noqa` everywhere? This makes the example less readable for people not used to specifics of python so I'd rather not have them.

Reply:
You can just remove the examples files from the flake8 check.

Reply:
Agreed